frontend/components/Blocks/SectionBlock/CostBenefitModal/CostBenefitModal.tsx

```diff
@@ -1,15 +1,51 @@
-import KostenBatenChart from "@/components/Charts/KostenBatenChart";
+import { useState, useEffect } from "react";
+import CostBenefitChart from "@/components/CostBenefit/CostBenefitChart";
+import CostBenefitDetail from "@/components/CostBenefit/CostBenefitDetail";
 import { Tab } from "@headlessui/react";
 import { XMarkIcon } from "@heroicons/react/24/outline";
 import classNames from "classnames";
-import CostBenefitTable from "@/components/Charts/CostBenefitTable";
+import {
+  getHolonDataSegments,
+  getHolonDataSegmentsDetail,
+  getHolonGraphColor,
+} from "../../../../api/holon";
```

Comment:
```suggestion
} from "@/api/holon";
```

Dit werkt volgens mij ook en zorgt ervoor dat de imports niet stuk gaan als het component verplaatst wordt.

Reply:
I tried changing this, but the test fails